### ISSUE TITLE
Failed to spawn error

### ISSUE THREAD

**Issue by brandon-collins** [tststs/atom-ternjs#111]
I'm using atom-ternjs version 0.7.1

Atom used to start up without error but now I'm getting an error and it seems to be a result of the above mentioned plugin version.

Here is the atom error message:
Failed to spawn command /Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper. Make sure /Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper is installed and on your PATH

Not sure what other information I can provide to be helpful.

**Comment by ivarprudnikov** [comment]
Have Atom `1.0.10` on OSX tried to install Atom Tern for the first time. And got the same error as described above.

More details from dev console:

```
Window load time: 2835ms [index.js:46]
Uncaught BufferedProcessError: Failed to spawn command `/Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper`. Make sure `/Applications/Atom.app/Contents/Frameworks/Atom Helper.app/Contents/MacOS/Atom Helper` is installed and on your PATH [/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:260]
```
![atom tern error screenshot](https://cloud.githubusercontent.com/assets/1052642/9628498/39667626-5163-11e5-9365-73657ea25876.png)

**Comment by ivarprudnikov** [comment]
duplicate of #107

**Comment by jordanbtucker** [comment]
Sorry, referenced the wrong PR. #144 fixes this (probably), #145 does not.
